cubeviz/image_viewer.py

```diff
@@ -516,6 +591,9 @@ def update_slice_index(self, index):
         if self.is_contour_active:
             self.draw_contour()
 
+        if self._stats_axes is not None:
+            self.update_stats()
+
     def fast_draw_slice_at_index(self, index):
         """
```

Comment:
The stats will not update in `fast_draw_slice_at_index`. Does this work well when you slide?

Reply:
The box doesn't disappear, which is cool! Personally I think it's fine to update the stats only when the slider is released. It might not be that much of a performance hit to do it while sliding, but it doesn't seem necessary.

Reply:
agreed